depthai_ros_driver/include/depthai_ros_driver/visibility.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
```

Comment:
Needed? Seems to not be used as of right now. Also if there are no specific reasons to hide some parts from being exported / visible, I'd suggest (for Windows) to just rely on: `set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON CACHE BOOL "")`